core/src/table/table.module.css

```diff
@@ -8,7 +8,11 @@ in native HTML */
 	/* Avoid actually "separate" the borders */
 	border-spacing: 0;
 }
+.container-fill {
+	/* To make full 100% width of table */
```

Comment:
"To make the table takes 100% of its container width" would be more grammatically correct